.betterer.results

```diff
@@ -2931,12 +2931,6 @@ exports[`better eslint`] = {
     "public/app/features/live/index.ts:5381": [
       [0, 0, 0, "Do not use any type assertions.", "0"]
     ],
-    "public/app/features/logs/components/log-context/LogRowContextProvider.tsx:5381": [
```

Comment:
yay 🤩

Reply:
Improving a feature and 
![image](https://user-images.githubusercontent.com/8092184/231768833-a8618a98-f106-436e-8222-ef85793dc9c2.png)
is nice, right? 😆

Reply:
yes, yes, yes